app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMessage } from '../redux/greetingsReducer';
+import axios from 'axios';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const message = useSelector((state) => state.greeting);
+
+  async function fetchMessage() {
+    await axios.get('api/messages').then((response) => {
+      dispatch(getMessage(response.data.greeting));
+    });
+  }
+
+  useEffect(() => {
+    fetchMessage();
+  }, []);
+
+  return (
+    <div>
+      <h1>{message}</h1>
```

Comment:
- [ ] Despite installing all dependencies and setting up the app correctly, nothing is displayed when I visit your app. 

  ![lod](https://user-images.githubusercontent.com/81468675/191291312-4fe1bdd7-d0c7-41e5-8ec9-247bbffee94e.jpg)

Reply:
Hi @julie-ify I solved the issue, and now my app is working perfectly